builder/googlecompute/config.go

```diff
@@ -73,7 +73,13 @@ func NewConfig(raws ...interface{}) (*Config, []string, error) {
 	}
 
 	if c.ImageName == "" {
-		c.ImageName = "packer-{{timestamp}}"
+		img, err := interpolate.Render("packer-{{timestamp}}", nil)
+		if err != nil {
+			panic(err)
```

Comment:
any reason you favor a `panic` here instead of returning `nil, nil, err` like line 59?